server/session/manager.go

```diff
@@ -2,53 +2,88 @@ package session
 
 import (
 	"context"
+	"errors"
+	"github.com/google/uuid"
 	"github.com/gorilla/websocket"
 	"github.com/rs/zerolog/log"
+	"platform-backend/models"
 	"platform-backend/server/api"
 	"sync"
 )
 
-type Manager struct {
+type ManagerImpl struct {
```

Comment:
Maybe need to make this a repository? And create "localstorage repository" for session manager as for other models+usecases

Reply:
moved